### PR TITLE
feat: List accounts within a managed group

### PR DESCRIPTION
**Pull request by gsusmi**
managed-group-accounts

### CODE REVIEW

File: addons/core/translations/resources/en-us.yaml [hunk 1]
```diff
@@ -362,3 +362,6 @@ managed-group:
     welcome:
       title: Welcome to Managed Groups
       description: Create and manage managed groups for an auth method.
+    accounts:
+      title: Welcome to Accounts
+      description: No accounts in this managed group.
```
Comment (calcaide) [inline addons/core/translations/resources/en-us.yaml:367]: This shouldn't be part of empty state?

**Review by calcaide — APPROVED** [review]
LGTM!! Thank you for the work

**Comment by calcaide** [comment]
Updated PR to refactor `accounts` for `members` within `managed-groups`. Added breadcrumbs

Screen Shot 2021-11-12 at 10 54 15 AM

File: addons/core/translations/resources/en-us.yaml (hunk 1 above)
Comment (gsusmi) [reply addons/core/translations/resources/en-us.yaml:367]: This is used in a new `members` tab, thus isn't an empty state for managed group.

File: addons/core/translations/resources/en-us.yaml [hunk 2]
```diff
@@ -362,3 +362,9 @@ managed-group:
     welcome:
       title: Welcome to Managed Groups
       description: Create and manage managed groups for an auth method.
+    member:
+      title: Welcome to Accounts
```
Comment (gsusmi) [inline addons/core/translations/resources/en-us.yaml:366]:
```suggestion
      title: Welcome to Members
```

File: addons/core/translations/resources/en-us.yaml [hunk 3]
```diff
@@ -362,3 +362,9 @@ managed-group:
...
+      description: No accounts in this managed group.
```
Comment (gsusmi) [inline addons/core/translations/resources/en-us.yaml:367]:
```suggestion
      description: No members in this managed group.
```

File: addons/core/translations/resources/en-us.yaml [hunk 4]
```diff
@@ -362,3 +362,9 @@ managed-group:
...
+member:
+  title: Member
+  title_plural: Members
```
Comment (gsusmi) [inline addons/core/translations/resources/en-us.yaml:370]: Thoughts on nesting this under managed-group? `target` resource is a good example of this.

File: ui/admin/app/controllers/scopes/scope/auth-methods/auth-method/managed-groups/members.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
 
-export default class ScopesScopeAuthMethodsAuthMethodManagedGroupsNewController extends Controller {
+export default class ScopesScopeAuthMethodsAuthMethodManagedGroupsMembersController extends Controller {
```
Comment (gsusmi) [inline ui/admin/app/controllers/scopes/scope/auth-methods/auth-method/managed-groups/members.js:4]: This file needs to be nested under `../auth-method/managed-groups/managed-group/member.js`

File: addons/core/translations/resources/en-us.yaml (hunk 4 above)
Comment (calcaide) [reply addons/core/translations/resources/en-us.yaml:370]: Definitely including this, makes all the sense :)

File: ui/admin/app/controllers/scopes/scope/auth-methods/auth-method/managed-groups/members.js (hunk 5 above)
Comment (calcaide) [reply ui/admin/app/controllers/scopes/scope/auth-methods/auth-method/managed-groups/members.js]: Updated. Good catch!

**Pull request merged**